app/javascript/components/mockApartments.js

```diff
@@ -0,0 +1,26 @@
+let apartments = [
+    {
```

Comment:
You'll want to make sure your mock apartments in the js file have ids and user_ids

Reply:
Thanks! Will implement.